### ISSUE TITLE
Bridge Troll users can opt out of chapter announcement emails

### ISSUE THREAD

**Issue by lilliealbert** [railsbridge/bridge_troll#442]
Next to the checkbox on the Settings page that says "Allow Bridge Troll to send you an email when a new event is posted in one of those regions", add a checkbox that says "Allow Bridge Troll to send you chapter announcement emails"

**Comment by kariabancroft** [comment]
I'd like to work on this one

**Comment by lilliealbert** [comment]
@kariabancroft Do you still plan on working on this? If not, could you let us know?

**Comment by kariabancroft** [comment]
@lilliealbert Yes! I think I got most of it working a little while ago. It will just take me a little bit to try to remember where I'm at again. 

I'm pretty sure I was working through the option we discussed in [this](https://groups.google.com/forum/#!topic/bridge-troll/H_24VYpov60) thread on the google group so I just want to double check that's where i'm at.

**Comment by lilliealbert** [comment]
Cool! @tjgrathwell do you have any opinions on changing this feature to region announcement emails rather than chapter announcement emails, since chapter membership isn't currently an actual thing in the app? (More background here: https://groups.google.com/forum/#!topic/bridge-troll/H_24VYpov60)
